src/transformers/utils/hub.py

```diff
@@ -1081,6 +1095,43 @@ def extract_info_from_url(url):
     return {"repo": cache_repo, "revision": revision, "filename": filename}
 
 
+def create_and_tag_model_card(
+    repo_id: str,
+    tags: Optional[List[str]] = None,
+    token: Optional[str] = None,
+    ignore_metadata_errors: bool = False,
```

Comment:
@Wauplin @amyeroberts fine to assume that `repo_type` will always be set to `"model"` in `ModelCard.load()` (inspecting at that methods signature)

Reply:
Yes definitely, `repo_type="model"` is implicit when using `ModelCard`.
(the parameter exists because `ModelCard` inherits from the generic `RepoCard`. Not a perfect design-choice but it's fine :) )

Reply:
ok perfect! let's keep it like that then